src/views/Ifos/hooks/v2/useGetPublicIfoData.ts

```diff
@@ -131,11 +130,45 @@ const useGetPublicIfoData = (ifo: Ifo): PublicIfoData => {
       endBlockNum,
       numberPoints: numberPoints ? numberPoints[0].toNumber() : 0,
     }))
-  }, [address, currentBlock, releaseBlockNumber])
+  }, [currentBlock, releaseBlockNumber, ifoCalls])
+
+  const fetchInActiveIfoData = useCallback(async () => {
```

Comment:
Same question as in v1, do we actually make less calls here? Seems like just a bit of saved performance with blocks/progress calculations. 🤔

Reply:
Yes, it only depends on ifoCalls which depends on address that means we will only call it once.